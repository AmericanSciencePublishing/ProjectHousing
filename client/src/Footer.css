.footer {
  padding: 2em 2em;
  margin-top: 2em;
  background-color: rgb(51, 51, 51);
  color: white;
}

@media screen and (max-width: 500px){
    .upper{
        text-align:center;
    }

    .upper a{
        padding:0 0.5em;
        color:white;
	font-size:10px;
    }
    
}

@media screen and (min-width: 500px){
    .upper{
	text-align:center;
    }
    
    .upper a{
	padding:0 2em;
	color:white;
	font-weight:bold;
    }
}

@media screen and (max-width: 500px){
    .lower{
    }

    .lower_copyright{
        text-align:center;
    }

    .lower_icons{
	text-align:center;
    }
    .lower_icons a{
        padding: 0 5px;
    }
}

@media screen and (min-width: 500px){
    .lower{
	display:flex;
    }

    .lower_copyright{
	flex:1;
	text-align:left;
    }

    .lower_icons{
	flex:1;
	text-align:right;
    }

    .lower_icons a{
	padding-right:1em;
    }
}

.fa-facebook-square:hover {
    color: #3b5998;
}

.fa-twitter:hover {
    color: #1dcaff;
}

.fa-weixin:hover {
    color: #09b83e;
}

.lower p{
    font-size: 1em;
}

.lower a {
    color:white;
}


