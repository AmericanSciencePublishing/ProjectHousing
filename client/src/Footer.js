import React from 'react';
import './css/Footer.css';
import Mailto from './Mailto';

const Footer = () => {

  return (
      <div className="footer">
	<div className="upper">
	  <a href="/about" target='_blank'>ABOUT US </a>
	  <Mailto email="szhao31@hawk.iit.edu" obfuscate={true}>CONTACT US
	    </Mailto>
	  <Mailto email="szhao31@hawk.iit.edu" obfuscate={true}> SUPPORT
          </Mailto>
	  <a href="/terms" target='_blank'>TERMS</a>
	  <a href="/terms" target='_blank'>PRIVACY</a>

	</div>
	<hr />
	<div className="lower">
	  <div className="lower_copyright">
	    <p>Copyright &copy;2018 All Rights Reserved</p>
	  </div>
	  <div className="lower_icons">
	    <a href="http://www.facebook.com" target='_blank' rel="noopener noreferrer"> <i className="fa fa-facebook-square fa-2x" aria-hidden="true"></i></a>
	    <a href="http://www.twitter.com" target='_blank' rel="noopener noreferrer"> <i className="fa fa-twitter fa-2x" aria-hidden="true"></i></a>
	    <a href="http://http://www.wechat.com/en/" target='_blank' rel="noopener noreferrer"> <i className="fa fa-weixin fa-2x " aria-hidden="true"></i></a>
	    </div>
	</div>
      </div>

  );
};

export default Footer;
