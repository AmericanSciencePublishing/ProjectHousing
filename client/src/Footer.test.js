import React from 'react';
import { shallow } from 'enzyme';

import Footer from './Footer';

it('renders footer without crashing', () => {
  const wrapper = shallow(<Footer />);

  expect(wrapper).toMatchSnapshot();
})
