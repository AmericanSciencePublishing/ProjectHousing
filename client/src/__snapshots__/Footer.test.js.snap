// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": <p>
        Copyright ©2017 All Rights Reserved
</p>,
      "className": "footer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": "Copyright ©2017 All Rights Reserved",
      },
      "ref": null,
      "rendered": "Copyright ©2017 All Rights Reserved",
      "type": "p",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": <p>
          Copyright ©2017 All Rights Reserved
</p>,
        "className": "footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Copyright ©2017 All Rights Reserved",
        },
        "ref": null,
        "rendered": "Copyright ©2017 All Rights Reserved",
        "type": "p",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
